crates/astria-proto/src/native/sequencer/v1alpha1.rs

```diff
@@ -1001,115 +1006,125 @@ impl Display for Address {
     }
 }
 
-#[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord)]
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash, PartialOrd, Ord)]
 #[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
 #[cfg_attr(feature = "serde", serde(transparent))]
-pub struct ChainId {
+pub struct RollupId {
```

Comment:
@noot @joroshiba I wanted to especially flag this: I renamed chain ID to rollup ID everywhere.